src/main/java/hudson/plugins/parameterizedtrigger/BuildTrigger.java

```diff
@@ -96,7 +99,19 @@ public void buildDependencyGraph(AbstractProject owner, DependencyGraph graph) {
 		for (BuildTriggerConfig config : configs) {
 			List<AbstractProject> projectList = config.getProjectList(owner.getParent(), null);
 			for (AbstractProject project : projectList) {
-				ParameterizedDependency.add(owner, project, config, graph);
+                            if (config.triggerFromChildProjects() && owner instanceof ItemGroup) {
+                                ItemGroup<Item> parent = (ItemGroup) owner;
+                                for (Item item : parent.getItems()) {
+                                    if(item instanceof AbstractProject){
```

Comment:
Thank you for all response, I put it into my code. I am afraid that only reworking ParametrizedDependency does not help.... https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/DependencyGraph.java#L394